docs/user/rstudio/ide/guide/publish/connecting.qmd

```diff
@@ -28,21 +34,24 @@ install.packages("rsconnect")
 
 ![Global options menu](images/connecting-1.png){fig-alt="Global options menu"}
 
-You'll need to connect RStudio to the remote hosting environment (e.g. Posit Connect, Shinyapps.io, RPubs.com) and authorize RStudio to use your account. To add a publishing account to RStudio, press **Tools** in the main menu, then **Global Options\...**.
+You'll need to connect RStudio to the remote hosting environment (e.g. Posit Connect, Shinyapps.io, Posit Cloud, RPubs.com) and authorize RStudio to use your account. To add a publishing account to RStudio, press **Tools** in the main menu, then **Global Options\...**.
```

Comment:
So RPubs.com actually doesn't connect an account to RStudio the way these other services do (like you cannot connect with Tools > Global Options > Publishing, so perhaps it's worth removing it from the parenthetical.

Reply:
removed rpubs